File: src/client/src/app/models/portfolio.ts
import { DataTableRow } from ".";
import { Guid } from "guid-typescript";

export interface Portfolio {
    id: Guid
    name: string;
    sum: number;
}

export interface PortfolioRow extends DataTableRow {
    id: string,
    name: string;
    sum: number;
}

File: src/server/InvestmentApp-Server/Models/Portfolio.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace InvestmentApp.Models;

public class Portfolio
{
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public Guid Id { get; set; }

    public string Name { get; set; }

    public int Sum { get; set; }
}
